sdk/services/prices.ts

```diff
@@ -187,11 +187,12 @@ export default class PricesService {
 		);
 		const prices = (response ? Object.values(response).flat() : []) as SynthPrice[];
 
+		//TODO: remove this once the rates are fixed
 		return prices.map((price) =>
 			price.synth === MarketAssetByKey[FuturesMarketKey.sPEPEPERP]
-				? { ...price, rate: wei(price.rate.toString()).div(1e10) }
+				? { ...price, rate: wei(price.rate.toString()).div(1e10).toString() }
```

Comment:
Good catch looks like this is an important one to get in a hotfix

Reply:
Yes. Agree. The error is also very annoying in the dev.

Reply:
I will also just try and fix on the subgraph now